arclightning_backend/src/main.rs

```diff
@@ -0,0 +1,79 @@
+extern crate futures;
+extern crate hyper;
+#[macro_use]
+extern crate serde_json;
+
+use futures::future;
+use hyper::rt::{Future, Stream};
+use hyper::service::service_fn;
+use hyper::{Body, Method, Request, Response, Server, StatusCode};
+
+// after this concept is further understood, will switch to 'Either'
```

Comment:
what does this mean?

Reply:
From the comment in the source code: https://github.com/hyperium/hyper/blob/master/examples/echo.rs